src/main/java/it/gov/pagopa/payment/dto/mapper/TransactionCreationRequest2TransactionInProgressMapper.java

```diff
@@ -31,21 +33,21 @@ public TransactionInProgress apply(
         .effectiveAmount(CommonUtilities.centsToEuro(transactionCreationRequest.getAmountCents()))
         .amountCurrency(PaymentConstants.CURRENCY_EUR)
         .merchantFiscalCode(merchantDetail.getFiscalCode())
-        .idTrxIssuer(transactionCreationRequest.getIdTrxIssuer())
+        .idTrxIssuer(idTrxIssuer)
         .initiativeId(transactionCreationRequest.getInitiativeId())
         .initiativeName(merchantDetail.getInitiativeName())
         .businessName(merchantDetail.getBusinessName())
         .mcc(transactionCreationRequest.getMcc())
         .vat(merchantDetail.getVatNumber())
-        .trxDate(transactionCreationRequest.getTrxDate())
-        .trxChargeDate(transactionCreationRequest.getTrxDate())
+        .trxDate(elaborationTrxDate)
+        .trxChargeDate(elaborationTrxDate)
```

Comment:
don't ask the elaborationTrxDate as parameter!
it's a mapper logic to fill it with the timestamp when creating